mandatory/3-playing-computer.js

```diff
@@ -17,7 +17,7 @@
 
 let x = 2;
```

Comment:
Can you please answer the questions above? The answers should be put underneath each question